### PR TITLE
Support actions

### PR DESCRIPTION
**Pull request by realstraw**

### CODE REVIEW

**Review by damianxu88 — APPROVED** [review]
LGTM

File: orchard-ws/app/models/WorkflowRequest.scala [hunk 1]
```diff
@@ -34,13 +35,24 @@ object WorkflowRequest {
     activityType: String,
     activitySpec: JsValue,
     resourceId: String,
-    maxAttempt: Int
+    maxAttempt: Int,
+    onSuccess: Option[Seq[String]],
```
Comment (anniezhang77) [inline orchard-ws/app/models/WorkflowRequest.scala:39]: what is stored in this Seq[String]?

Comment (realstraw) [reply orchard-ws/app/models/WorkflowRequest.scala:39]: the action IDs

File: orchard-ws/app/models/WorkflowRequest.scala [hunk 2]
```diff
@@ -34,13 +35,24 @@ object WorkflowRequest {
...
+    onFailure: Option[Seq[String]]
+  )
+
+  case class Action(
+    id: String,
+    name: String,
+    actionType: String,
```
Comment (anniezhang77) [inline orchard-ws/app/models/WorkflowRequest.scala:46]: actionType is sns, etc?

**Pull request merged**